lib/topic.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+import { config } from './config';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
+};
+
+export const deleteKafkaTopic = async function (page: Page, name: string) {
+  const instanceLinkSelector = page.getByText(name);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Delete').click();
+  await page.getByLabel('Type DELETE to confirm:').click();
+  await page.getByLabel('Type DELETE to confirm:').fill('DELETE');
+  await page.getByTestId('modalDeleteTopic-buttonDelete').click();
+  await page.getByTestId('mk--instance__drawer').getByText('Kafka Instances').click();
+  await page.waitForURL(config.startingPage + '/application-services/streams/kafkas');
```

Comment:
I think that this line is not needed.

Reply:
all lines there are necessary, what's more they come from debugging output and perform the necessary actions to delete a topic

Reply:
related to the answer above, no, this is not necessary with the current design of the tests.

Reply:
removed